index.js

```diff
@@ -0,0 +1,80 @@
+const apiUrl = 'https://ghibliapi.herokuapp.com';
+let localMovieReviewStorage = {};
+
+document.addEventListener('DOMContentLoaded', () => {
+	seedSelectBox();
+	let selectBox = document.querySelector('.movie-titles');
+	selectBox.addEventListener('change', getMovieInfo);
+	let form = document.querySelector('.user-review');
+	form.addEventListener('submit', makeNewReview);
+});
+
+async function seedSelectBox(){
+	let selectBox = document.querySelector('.movie-titles');
+	let movies = await axios.get(`${apiUrl}/films`);
```

Comment:
The variable you named `movies` actually contains the `response` object from `axios`. A better name would have been `response` and then create another variable `let movies = response.data`